tests/components/google_assistant/test_trait.py

```diff
@@ -845,38 +845,20 @@
                                            None)
 
     trt = trait.LockUnlockTrait(hass,
-                                State('lock.front_door', lock.STATE_UNLOCKED),
+                                State('lock.front_door', lock.STATE_LOCKED),
                                 PIN_CONFIG)
 
     assert trt.sync_attributes() == {}
 
     assert trt.query_attributes() == {
-        'isLocked': False
+        'isLocked': True
     }
 
     assert trt.can_execute(trait.COMMAND_LOCKUNLOCK, {'lock': True})
 
     calls = async_mock_service(hass, lock.DOMAIN, lock.SERVICE_LOCK)
 
-    # No challenge data
```

Comment:
we should test that we raise challenge needed?

Reply:
That's the point of the PR, only challenge for a pin to unlock the lock or (partially)open the cover, not to lock it or close it.